internal/parse/parser_test.go

```diff
@@ -41,155 +41,95 @@ var tests = []struct {
 }, {
 	"quoted output expression",
 	"SELECT p.* AS &Person.*, '&notAnOutputExpresion.*' AS literal FROM t",
-	"ParsedExpr[BypassPart[SELECT p.* AS &Person.*, ] " +
-		"BypassPart['&notAnOutputExpresion.*'] " +
-		"BypassPart[ AS literal FROM t]]",
+	"ParsedExpr[BypassPart[SELECT p.* AS &Person.*, ] BypassPart['&notAnOutputExpresion.*'] BypassPart[ AS literal FROM t]]",
```

Comment:
The output parse isn't actually merged yet! These tests were originally designed for the whole parse which we wrote at once but have submitted as separate PRs (though its all changed a bit since then).

Reply:
Ah, I understand. Thanks!